tests/e2e/graphql/graphql_crud.py

```diff
@@ -0,0 +1,70 @@
+import sys
+
+import pytest
+from graphql_server import *
+
+
+def test_create_query(query_server):
+    create_node_query(query_server)
```

Comment:
I think `create_node_query` should just create node and then this function should do assertions

Reply:
As you can see, this function is being used as a building block for the subsequent tests, this would mean that either I have to assert the correct response after every creation in every test, or I just do not assert in the other tests, which feels a bit odd. I would prefer asserting in the function.
What do you think?

Reply:
Yes currently yes. But I think that the logic is a bit wrong: because currently you have a server which creates example but the first extension of this would be to let clients send example they want (and I think this is how it should be even ATM, graphql_crud should send object which server needs to work with). So I think that current way doesn't really respect open-closed principle for extension.

Reply:
Also, by testing things in graphql_server you don't have clear boundary between what are tests and what are just "helper" classes

Reply:
Modified these, I removed the checks from the helper functions and the bare functionality of creating vertices is done in the CREATE tests themselves.